internal/logger/new_logger.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: remove these methods/file after slog support is added to jacobsa/fuse.
```

Comment:
Please rename this method to legacy_logger and mention that it is required to support jacobsa/fuse loggers and will be removed after slog support is adde.d